test/steps/data.go

```diff
@@ -111,6 +119,9 @@ func (data *Data) AfterScenario(pickle *messages.Pickle, err error) error {
 	logScenarioDuration(data)
 	deleteTemporaryExamplesFolder(data)
 
+	// Remove this once new Strimzi version with https://github.com/strimzi/strimzi-kafka-operator/issues/3092 fix is released
+	data.zookeeperMonitoring <- true
```

Comment:
For me, "true" means to start monitoring zookeeper and as far I understand below. when "true" will actually stop monitoring it.